tests/includes/API/ApiExceptionTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * Created by PhpStorm.
+ * User: Matthew
+ * Date: 5/21/16
+ * Time: 22:35
+ */
+
+use \Waca\API\ApiException;
+
+class ApiExceptionTest extends PHPUnit_Framework_TestCase
+{
+	private $message;
+	private $exception;
```

Comment:
`/** @var ApiException */` will help code analysis tools work here if you stick it before `private $exception;`